src/styles/blocks/header.scss

```diff
@@ -0,0 +1,10 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
```

Comment:
```suggestion
  background: $white_accent;
```

Reply:
Still not fixed

Reply:
accidentally pressed the button re-request